pygmt/src/grdgradient.py

```diff
@@ -0,0 +1,118 @@
+"""
+grdgradient - Create a gradient from a grid.
+"""
+
+import xarray as xr
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import (
+    GMTTempFile,
+    args_in_kwargs,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    A="azimuth",
+    D="direction",
+    E="radiance",
+    G="outgrid",
+    R="region",
+    V="verbose",
+)
```

Comment:
Add common alias interpolation (-n).

```suggestion
    V="verbose",
    n="interpolation",
)
```

Reply:
I was planning on adding optional parameters in a subsequent pull request; I'll add `n` but will save `N`, `Q`, and `S` for a future PR.